src/main/java/nl/tudelft/contextproject/model/entities/control/BunnyAI.java

```diff
@@ -37,15 +37,18 @@ public BunnyAI() {
 		this.player = game.getPlayer();
 		this.entities = game.getEntities();
 	}
-	
+
 	@Override
 	public void move(float tpf) {
+		if (owner.getSpatial() != null) {
+			owner.getSpatial().lookAt(player.getSpatial().getWorldTranslation(), Vector3f.UNIT_Y);
+		}
```

Comment:
You should moce this code to line 54 (below findTarget()) and change some things so it looks like this: 
`Spatial sp = owner.getSpatial();
		if (sp != null) {
			sp.lookAt(target.getLocation(), Vector3f.UNIT_Y);
			sp.rotate(0, (float) Math.toRadians(-90), 0);
		}`
The bunny is otherwise rotated incorrect and it now always faces its target.
I would also suggest to scale the bunny at creation time with a factor 2, as he is very small now